### PR TITLE
Blog post: Alpha release

### PR DESCRIPTION
**Pull request by czosel**
Alright, [here's a first draft](https://github.com/czosel/plugin-php/blob/6cdb7f7942bd9e0716ef31f7b68f7fd31f69ea24/blog-post-0.1.md) - let me know what you think :smile:

### CODE REVIEW

**Comment by evilebottnawi** [comment]
@vjeux can we post news and release in official prettier site?

**Comment by vjeux** [comment]
@evilebottnawi yeah of course! Please send a PR there and I'll ship it and tweet about it

**Comment by evilebottnawi** [comment]
@czosel english is your native language?

**Comment by czosel** [comment]
@evilebottnawi No! Please let me know if you see something that sounds strange :wink:

**Comment by evilebottnawi** [comment]
@czosel English also is not my native language, we should found who have English as native language for post/tweet and etc (grammatical spell and other mistakes) :smile:

**Comment by vjeux** [comment]
I wouldn't worry about having proper English too much, the important is that we get the message across, if there are some mistakes it's not the end of the world (English is also not my first tongue :P)

**Comment by hawkrives** [comment]
The draft read fine to me; I wouldn't have guessed that it wasn't your first language. 

I'd be happy to proof anything in the future, too.

File: blog-post-0.1.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+* A parser, which turns your source code into an abstract syntax tree (AST). Since Prettier is written in JavaScript, we're using [php-parser](https://github.com/glayzzle/php-parser), which is a PHP parser written in JS.
+
+* A printer, that turns the AST into a special format called `doc` that Prettier requires to do it's magic. It is basically your source code plus some information about where linebreaks can be added, how it should be indented, and so on.
+
+While we could benefit from existing work on the parsing side, the printer had to be developed from scratch to support all of PHP's various AST node types. Our first alpha release today means that we are confident that the printer can re-print your code without breaking it.
+
+## Philosophy
+
+When building a code formatter, it can be tempting to add many options to account for all the different code styles there are. The PHP plugin sticks to Prettier's opinionated philosophy and doesn't add any new options. Instead, we identiefied [PSR-1, PSR-2 and PSR-12](https://www.php-fig.org/psr/) as the most popular coding styles in the PHP community and decided to implement those.
```
[the remark this answers is not in the document]
Comment (czosel) [reply blog-post-0.1.md]: :+1:

File: blog-post-0.1.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
+# Prettier for PHP 0.1: First alpha release :tada:
+
+After more than 200 merged pull requests since mid December 2017, we're happy to announce the first alpha release of Prettier for PHP. In this blog post, we'd like to give a short overview of how the plugin works, it's philosophy, and what to expect in the future.
```
[the remark this answers is not in the document]
Comment (czosel) [reply blog-post-0.1.md]: The idea is to post it to here: https://prettier.io/blog/ - so it's definitely a good idea to link to our repo.

**Comment by mgrip** [comment]
🎉

I also think we should make sure https://github.com/prettier/plugin-php/issues/356 is addressed before launching. Not sure if anything needs to change or not, but we should put some thought into it to make sure its super clear how people can easily install this and try it out

File: blog-post-0.1.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    Since every new release that changes the way things are printed requires a complete re-print of your codebase, the visual quality of our output has to very good before we can make a stable release.
+
+3.  Integrations
+
+    Today, you can try the plugin from the command line by following the instructions in the README. Plugin support in your favorite editor probably won't work yet, though (see the related issues for [vs code](https://github.com/prettier/prettier-vscode/issues/395), [atom](https://github.com/prettier/prettier-atom/issues/395) and [vim](https://github.com/prettier/vim-prettier/issues/119)). Also, the PHP plugin is not integrated in Prettier's [playground](https://prettier.io/playground/) yet.
+
+Are you excited about Prettier for PHP? Then we'd love you to help us reach our first stable release by
+
+* Trying it on your code base
+* Reporting any issues you find with it ([here](https://github.com/prettier/plugin-php/issues))
```
Comment (mgrip) [inline blog-post-0.1.md]: Maybe add a bullet for "Contributing!"?

https://github.com/prettier/plugin-php/labels/good%20first%20issue

(and we should be better about using that label 😆 )

Comment (czosel) [reply blog-post-0.1.md]: Good idea :+1:

**Comment by czosel** [comment]
https://github.com/prettier/prettier/pull/4289

**Pull request closed without merging**